doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The number of RPCs is configured by the experimental flag, `soak_iterations`.
+
+This test is configurable via a few different command line flags:
+
+* `soak_iterations`: controls the number of RPCs to perform.
+
+* `soak_max_failures`: an inclusive upper limit on the number of RPC failures
+  that should be tolerated (i.e. after which the test process should
+  still exit 0). A failure is considered to be either a non-OK status or an RPC
+  whose latency exceeds `soak_per_iteration_max_acceptable_latency_ms`.
+
+* `soak_per_iteration_max_acceptable_latency_ms`: an inclusive upper limit
+  on the latency of a single RPC in order for that RPC to be considered
+  successful.
+
+* `soak_overall_timeout_seconds`: the overall number of seconds after which
+  the test should stop and fail if `soak_iterations` have not yet been
+  completed.
+
+The following suggestions are optional but encouraged to improve debuggability:
+
+* Implementations should avoid setting RPC deadlines and should instead
+  wait for each RPC to complete.
+
+* Implementations should log the number of milliseconds that each RPC takes.
```

Comment:
That's a lot of logging. If this is ever used in an automated fashion it seems it will spam the logs a ton, unless we are expecting to use smaller values of soak_iterations like "10".

Reply:
We have been running this test for C++ continuously, using up to 8K iterations. The log output can indeed be fairly big, but it hasn't really been a problem so far (e.g. test infra is able to handle the size of the resulting logs). The test's summary log being at the very bottom also helps to skip over these quickly at a glance.

What do you think about another flag to enable or disable these logs?

Tests with extremely large numbers of iterations could that way disable it.